Makefile

```diff
@@ -13,17 +13,23 @@ wasms_for_tests := wasm_for_tests/wasm_source
 # Paths for all the wasm templates
 wasm_templates := wasm/tx_template wasm/vp_template
 
+ifdef JOBS
```

Comment:
this can be set in your local cargo config https://doc.rust-lang.org/cargo/reference/config.html#configuration-format